server/debian.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:10-slim
+
+LABEL varion="2.0" \
+    maintainer="ESh4d0w, Markus Hadenfeldt, docker@teaspeak.de, h1dden-da3m0n" \
+    description="A simple TeaSpeak server running on debian 10 (amd64)"
+
+ARG TEASPEAK_VERSION="1.4.12"
+
+RUN	apt-get update && apt-get upgrade -qqy \
+    && apt-get install --no-install-recommends -qqy \
+         ca-certificates wget curl python libnice10 ffmpeg tzdata \
```

Comment:
For what is `tzdata`?

Reply:
`tzdata` allows the user to set a TZ variable via the `docker run` execution or in a `docker-compse` file, which allows the user to set the timezone within the container